### PR TITLE
Customer class

### PR DESCRIPTION
**Pull request by Wilsonator123**

### CODE REVIEW

**Review by khadir786 — APPROVED** [review]

File: Customer.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+        public void OpenIsaAccount()
+        {
+            if (_isaAccount == null)
+            {
+                // TODO : Create ISA Account
+                // _isaAccount = new IsaAccount();
+            }
+        }
+
```
Comment (perry-C) [inline Customer.cs:98]: Is this for testing purposes?

File: Customer.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        public static void Main()
```
Comment (perry-C) [inline Customer.cs:99]: Is this for testing?

**Comment by Wilsonator123** [comment]
@khadir786 its not finished yet :-1:

**Review by perry-C — CHANGES_REQUESTED** [review]
- should pull from main first, me and Khadir merged I few branches this morning so it will cause conflicts
- And also leave out the bin folder

**Comment by khadir786** [comment]
> * should pull from main first, me and Khadir merged I few branches this morning so it will cause conflicts
> 
>     * And also leave out the bin folder

ye maybe stash changes, then pull from main?

File: Customer.cs (hunk 2 above)
Comment (Wilsonator123) [reply Customer.cs:104]: Not actual testing but will be removed when finished

**Comment by Wilsonator123** [comment]
Might change name -> FirstName and LastName for validating IDs

@khadir786 @perry-C

File: Customer.cs (hunk 2 above)
Comment (perry-C) [reply Customer.cs:104]: np

**Comment by Wilsonator123** [comment]
> - should pull from main first, me and Khadir merged I few branches this morning so it will cause conflicts
> - And also leave out the bin folder
> - Otherwise looks good, nice touch on having the ToString() method for the customer class 

It's the Java in me

**Review by perry-C — APPROVED** [review]

**Pull request merged**